src/Microsoft.ApplicationInsights.AspNet/ExceptionTrackingMiddleware.cs

```diff
@@ -5,12 +5,12 @@
     using System;
     using System.Threading.Tasks;
 
-    public sealed class ApplicationInsightsExceptionMiddleware
+    public sealed class ExceptionTrackingMiddleware
```

Comment:
What do you think about term "Telemetry" instead of "Tracking" in the names of exception and request middleware?